notebooks/notebooks.md

```diff
@@ -0,0 +1 @@
+Place to store notebooks
```

Comment:
If the intent is that this is for users' notebook files, it doesn't belong in the project repository.

Reply:
Deleted